src/store/modules/nft.js

```diff
@@ -47,17 +46,15 @@ const mutations = {
   [TYPES.NFT_SET_USER_CLASSID_LIST_MAP](state, { address, nfts }) {
     Vue.set(state.userClassIdListMap, address, nfts);
   },
-  [TYPES.NFT_SET_USER_NFT_CLASS_FEATURED_SET_MAP](
+  [TYPES.NFT_SET_USER_NFT_CLASS_DISPLAY_STATE_SETS_MAP](
     state,
-    { address, classIdSet }
+    { address, nftDisplayStateSets }
   ) {
-    Vue.set(state.userNFTClassFeaturedSetMap, address, classIdSet);
-  },
-  [TYPES.NFT_SET_USER_NFT_CLASS_HIDDEN_SET_MAP](
-    state,
-    { address, classIdSet }
-  ) {
-    Vue.set(state.userNFTClassHiddenSetMap, address, classIdSet);
+    Vue.set(
+      state.userNFTClassDisplayStateSetsMap,
+      address,
+      nftDisplayStateSets
+    );
```

Comment:
We should explicitly define featuredClassIdSet and hiddenClassIdSet here, since we dont just want any arbitrary member and  they are expected everywhere

Reply:
I mean we should not directly store nftDisplayStateSets here, we should explicitly destruct it into the two member we need, and explicitly store the 2 member, such that it is clear that we want these two member in this mutations

Reply:
We can, or if you want to limit the diff in this PR to only hide/feature related code, maybe in later PRs